src/NetVar.h

```diff
@@ -280,8 +280,14 @@ extern zeek::RecordType* dns_dnskey_rr;
 [[deprecated("Remove in v4.1.  Perform your own lookup.")]]
 extern zeek::RecordType* dns_nsec3_rr;
 [[deprecated("Remove in v4.1.  Perform your own lookup.")]]
+extern zeek::RecordType* dns_nsec3param_rr;
```

Comment:
You don't need to add these ones here, since they're new for this work. They'd just get removed the next release anyways.

Reply:
Removed them from here.